src/DropdownMenu.js

```diff
@@ -1,12 +1,101 @@
 import PropTypes from 'prop-types';
-import React, { useContext, useRef, useEffect } from 'react';
-import { Popper } from 'react-popper';
-import usePrevious from '@restart/hooks/usePrevious';
+import { useContext, useRef } from 'react';
 
+import useCallbackRef from '@restart/hooks/useCallbackRef';
+import usePopper from './usePopper';
 import useRootClose from './useRootClose';
-import { useMergedRefs } from './utils/mergeRefs';
 import DropdownContext from './DropdownContext';
 
+export function useDropdownMenu(options = {}) {
+  // const prevProps = usePrevious(props);
```

Comment:
?

i guess this is all in `usePopper` now?

Reply:
yeah I think it's unneeded here now, but i need to try in RB to confirm